### PR TITLE
Add wireguard set example

### PR DESCRIPTION
**Pull request by dtoubelis**
Added an example of wireguard set operation.

### CODE REVIEW

File: Cargo.toml
Comment (Leo1003) [inline Cargo.toml]: The dependencies only used by examples or tests should be added as dev-dependencies instead.

**Review by Leo1003 — CHANGES_REQUESTED** [review]

File: Cargo.toml
Comment (dtoubelis) [reply Cargo.toml]: Right 👍 Updated.

**Review by Leo1003 — APPROVED** [review]
LGTM.
However, although I am the original contributor of this crate, I don't get the write access of the new repository. We still need someone w/ write access to approve this.

**Comment by dtoubelis** [comment]
Hmm, who else can approve it? @cathay4t?

**Review by cathay4t — APPROVED** [review]

**Comment by cathay4t** [comment]
@Leo1003 I have invite you with write access to this project. Sorry for missed that.

I am very sorry for the long wait. Approved. Will merge once CI fixed.

I have plan to improve my upstream involvement for rust-netlink projects in 2024, will not happen again.

**Pull request merged**
